lib/common/huf.h

```diff
@@ -111,6 +111,9 @@ HUF_PUBLIC_API size_t HUF_compress2 (void* dst, size_t dstCapacity, const void*
 #define HUF_WORKSPACE_SIZE_U32 (HUF_WORKSPACE_SIZE / sizeof(U32))
 HUF_PUBLIC_API size_t HUF_compress4X_wksp (void* dst, size_t dstCapacity, const void* src, size_t srcSize, unsigned maxSymbolValue, unsigned tableLog, void* workSpace, size_t wkspSize);
 
+/* The workspace must have alignment at least 4 and be at least this large */
+#define HUF_DECOMPRESS_WORKSPACE_SIZE (2 << 10)
```

Comment:
From some quick calculations, with `HUF_TABLELOG_MAX` set to `HUF_TABLELOG_ABSOLUTE_MAX`, ~1860 bytes is needed (so 7<<8 would not be enough).  with its current value, ~1500 bytes is needed. 

Do you think it's worth having the actual formula?